### PR TITLE
Adding draft agrawala paper

### PR DESCRIPTION
**Pull request by taylorgibson**

### CODE REVIEW

**Comment by rowanc1** [comment]
If we need to edit the checks that you want @taylorgibson, let me know. This is failing on a missing DOI and missing data availability statement. The second, maybe you don't want?

File: papers/morganton-2024-agrawala/myst.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    statement: Funding for this program is provided by the North Carolina School of Science and Mathematics, the University of North Carolina General Administration, and the General Assembly for the State of North Carolina.
+  error_rules:
+    - rule: doi-exists
+      severity: ignore
+      keys:
+        - Atr03
+        - terradesert
+        - jupyter
+        - sklearn1
+        - sklearn2
```
Comment (rowanc1) [inline papers/morganton-2024-agrawala/myst.yml:51]:
```suggestion
        - 'JMLR:v15:srivastava14a'
```

Comment (rowanc1) [reply papers/morganton-2024-agrawala/myst.yml:51]: If you add this, I can turn of the data statement in a sec!

**Comment by rowanc1** [comment]
Ok - now only the ORCID isn't there. Let me know if you want that to be optional this year.

**Comment by taylorgibson** [comment]
@rowanc1 Let's make that optional this year too. We've already decided to make that a requirement for submission next year!

**Comment by rowanc1** [comment]
Ok - those changes are implemented. 🚀

**Pull request merged**